WordPress/src/main/java/org/wordpress/android/ui/posts/mediauploadcompletionprocessors/BlockProcessor.java

```diff
@@ -129,6 +131,14 @@ String processBlock(String block) {
         return processBlock(block, false);
     }
 
+     final void addIdPropertySafely(JsonObject jsonAttributes, String idName, String remoteId) {
```

Comment:
Nitpick: We could use this function in the future to set integer properties, so I wonder if we could name it more generically. E.g.
```suggestion
     final void addIntPropertySafely(JsonObject jsonAttributes, String propertyName, String value) {
```